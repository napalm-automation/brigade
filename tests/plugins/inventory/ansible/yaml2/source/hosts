all:
    hosts:
        one.example.com:
        two.example.com:
            my_var: from_hostfile
        three.example.com:
            ansible_port: 5555
            ansible_host: 192.0.2.50
