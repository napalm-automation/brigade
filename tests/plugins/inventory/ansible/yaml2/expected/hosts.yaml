one.example.com:
  groups: []
three.example.com:
  brigade_host: 192.0.2.50
  brigade_ssh_port: 5555
  groups: []
two.example.com:
  groups: []
  my_var: from_hostfile
